themes/finna2/templates/RecordDriver/SolrQdc/core.phtml

```diff
@@ -135,21 +135,24 @@
         </div>
         <?php endif; ?>
 
-        <?php $publications = $this->driver->getPublicationDetails(); if (!empty($publications)): ?>
-        <div class="recordPublications">
-        <?php foreach ($publications as $field): ?>
-            <span property="publisher" typeof="Organization">
-            <?php $pubPlace = $field->getPlace(); if (!empty($pubPlace)): ?>
-              <span property="location"><?=$this->escapeHtml($pubPlace)?></span>
-            <?php endif; ?>
-            <?php $pubName = $field->getName(); if (!empty($pubName)): ?>
-              <span property="name"><?=$this->escapeHtml($pubName)?></span>
-            <?php endif; ?>
-            <?php $pubDate = $field->getDate(); if (!empty($pubDate)): ?>
-              <span property="datePublished"><?=$this->escapeHtml($pubDate)?></span>
-            <?php endif; ?>
-            </span>
-          <?php endforeach; ?>
+        <?php
+          $publications = $this->driver->getPublicationDetails();
```

Comment:
Tämä voisi olla samalla tavalla kuin descriptions alempana

Reply:
Juu, tiivistin kans sisällä olevat ku näytettäis alustavan tyhjinä jos ei arvoa ole.